hello_phoenix/ui/lib/ui/application.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Ui.Application do
```

Comment:
UiWeb?

Reply:
`UiWeb` doesn't have a `Application` because it's just the Phoenix-specific parts of the application. `Ui.Application` is the OTP application.